src/wallet/scriptpubkeyman.h

```diff
@@ -609,6 +609,7 @@ class DescriptorScriptPubKeyMan : public ScriptPubKeyMan
     bool CanGetAddresses(bool internal = false) const override;
 
     std::unique_ptr<SigningProvider> GetSolvingProvider(const CScript& script) const override;
+    std::unique_ptr<SigningProvider> GetSolvingProvider(const CScript& script, bool include_private) const { return GetSigningProvider(script, true); }
```

Comment:
`include_private` parameter is not being used.

```suggestion
    std::unique_ptr<SigningProvider> GetSolvingProvider(const CScript& script, bool include_private) const { return GetSigningProvider(script, include_private); }
```

Reply:
ups, thanks. Pushed.